### PR TITLE
fix: ptk completer renders new line as ^J

### PR DESCRIPTION
**Pull request by jnoortheen**
## For community
⬇️  **Please click the 👍 reaction instead of leaving a `+1` or 👍  comment**

### CODE REVIEW

File: xonsh/ptk_shell/completer.py [hunk 1]
```diff
@@ -98,11 +98,17 @@ def get_completions(self, document, complete_event):
...
+                    comp.description.replace(os.linesep, " ")
+                    if comp.description
+                    else None
+                )
                 yield Completion(
                     comp,
                     -comp.prefix_len if comp.prefix_len is not None else -plen,
                     display=comp.display or comp[pre:].strip("'\""),
-                    display_meta=comp.description or None,
+                    # ptk doesn't render newlines. This can be removed once it is supported.
```
Comment (daniel-shimon) [inline xonsh/ptk_shell/completer.py:110]: Shouldn't this comment be above the `replace` line?

File: xonsh/ptk_shell/completer.py [hunk 2]
```diff
@@ -98,11 +98,17 @@ def get_completions(self, document, complete_event):
         for comp in completions:
             # do not display quote
             if isinstance(comp, RichCompletion):
+                desc = (
+                    comp.description.replace(os.linesep, " ")
```
Comment (daniel-shimon) [inline xonsh/ptk_shell/completer.py:102]: Maybe `. ` (dot newline) so the sentences appear split?

Comment (jnoortheen) [reply xonsh/ptk_shell/completer.py:102]: putting dot would mean certain meaning. then we may want to title the new start ...

File: xonsh/ptk_shell/completer.py (hunk 1 above)
Comment (jnoortheen) [reply xonsh/ptk_shell/completer.py]: updated.

File: xonsh/ptk_shell/completer.py (hunk 2 above)
Comment (jnoortheen) [reply xonsh/ptk_shell/completer.py:103]: we can add some special characters like the return character. is that supported in most of the terminals? or we can have that in a setting ?

Comment (daniel-shimon) [reply xonsh/ptk_shell/completer.py:103]: I don't think this is a big deal
Just decide on something that looks nice

Comment (jnoortheen) [reply xonsh/ptk_shell/completer.py:103]: space is fine for now

**Review by daniel-shimon — APPROVED** [review]
Great!

**Pull request merged**